auth/src/main/java/org/openedx/auth/presentation/logistration/Logistration.kt

```diff
@@ -0,0 +1,207 @@
+                Column(Modifier.padding(bottom = 8.dp)) {
+                    Text(
+                        modifier = Modifier.padding(bottom = 10.dp),
+                        style = MaterialTheme.appTypography.titleMedium,
+                        text = stringResource(id = R.string.pre_auth_search_title),
+                    )
+                    SearchBar(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .height(48.dp),
+                        label = stringResource(id = R.string.pre_auth_search_hint),
+                        requestFocus = false,
+                        searchValue = textFieldValue,
+                        keyboardActions = {
+                            focusManager.clearFocus()
+                            onSearchClick(textFieldValue.text)
+                        },
+                        onValueChanged = { text ->
+                            textFieldValue = text
+                        },
+                        onClearValue = {
+                            textFieldValue = TextFieldValue("")
+                        }
+                    )
+                }
+
+                Text(
+                    modifier = Modifier
+                        .padding(bottom = 32.dp)
+                        .noRippleClickable {
+                            onSearchClick("")
+                        },
+                    text = stringResource(id = R.string.pre_auth_explore_all_courses),
+                    color = MaterialTheme.appColors.primary,
+                    style = MaterialTheme.appTypography.labelLarge
+                )
+
+                Spacer(modifier = Modifier.weight(1f))
+
+                Row {
+                    OpenEdXButton(
+                        width = Modifier
+                            .width(0.dp)
+                            .weight(1f),
+                        text = stringResource(id = R.string.auth_register),
+                        onClick = { onRegisterClick() }
+                    )
+
+                    OpenEdXOutlinedButton(
+                        modifier = Modifier
+                            .width(100.dp)
+                            .padding(start = 16.dp),
+                        text = stringResource(id = R.string.auth_sign_in),
+                        onClick = { onSignInClick() },
+                        borderColor = MaterialTheme.appColors.textFieldBorder,
+                        textColor = MaterialTheme.appColors.primary
+                    )
+                }
+            }
+        }
+    }
+}
+
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_NO)
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Preview(name = "NEXUS_9_Light", device = Devices.NEXUS_9, uiMode = Configuration.UI_MODE_NIGHT_NO)
+@Preview(name = "NEXUS_9_Night", device = Devices.NEXUS_9, uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Composable
+private fun SignInScreenPreview() {
```

Comment:
Please rename `SignInScreen`. nit: Also there are 2 different names on this screen, fragment is called `Logistration`, and main Composable is called `PreLoginScreen`